pcdsdevices/interface.py

```diff
@@ -382,6 +384,27 @@ def post_elog_status(self):
         elog.post(final_post, tags=['ophyd_status'],
                   title=f'{self.name} status report')
 
+    def screen(self):
+        """
+        Open a screen for controlling the device.
+
+        Default behavior is the typhos screen, but this method can
+        be overridden for more specialized screens.
+        """
+
+        if shutil.which('typhos') is None:
+            logger.error('typhos is not installed, ',
+                         'screen cannot be opened')
+            return
+
+        arglist = ['typhos', f'{self.name}']
+        logger.info(f'Opening typhos screen for {self.name}...')
+
+        # capture stdout and stderr
+        subprocess.Popen(arglist,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE)
```

Comment:
Piping stdout/stderr to where exactly? Did you by chance just want it to be dumped to `/dev/null`? (https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL)

Could `open_typhos_for_device` be a utility function?

Reply:
What were you thinking about for the `open_typhos_for_device` function?  Were you thinking of passing other typhos kwargs through?  I figured that if people wanted to open a typhos screen they would typically just open it from another shell session?

Reply:
I think what you have is just fine, really. I meant that "open a typhos screen for a device named X in a subprocess" is a generically reusable utility, such that creating something with an API like the following could be useful:

```python
def open_typhos_for_device(device_name: str, **some_options_here):
    ... # insert your .screen() functionality here
```

Reply:
I'm surprised there isn't already a typhos built-in for this